### PR TITLE
Update Jaeger Operator CHANGELOG.md

### PR DESCRIPTION
**Pull request by cokiengchiara**
Right now there is no text. 

- [x] Let's add some info.
- [ ] Update this announcement https://github.com/giantswarm/releases/pull/635/files

### CODE REVIEW

**Comment by cokiengchiara** [comment]
@ced0ps please write what was upgraded here. I couldn't figure it out from the commits. Thank you 🙏

**Comment by ced0ps** [comment]
@cokiengchiara done. should we still update the release announcement?

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,14 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [0.2.3] - 2021-02-26
 
+### Updated
+
+- Sync with upstream chart version --> 2.19.1
+
+### Changed
+
+- Collector service to support additionally support distributed deployment configuration
```
Comment (cokiengchiara) [inline CHANGELOG.md:18]:
```suggestion
- Collector service now supports configuration of distributed deployments
```
@ced0ps I don't understand this... is my suggested modification correct?

**Comment by cokiengchiara** [comment]
> @cokiengchiara done. should we still update the release announcement?

no need.

**Review by ced0ps — APPROVED** [review]

**Pull request merged**